packages/ember-codemod-remove-ember-css-modules/src/migration/ember-app/steps/shared/import-styles.ts

```diff
@@ -81,7 +94,9 @@ function importStylesInClass(file, data) {
   });
 
   // Append the styles import
+  // @ts-ignore: Assume that types from external packages are correct
```

Comment:
`ember-template-recast@6.1.4` didn't support TypeScript as well as I had hoped. The library [depends on `ASTv1` from `@glimmer/syntax`](https://github.com/ember-template-lint/ember-template-recast/blob/v6.1.4/src/index.ts#L2), while [`@glimmer/syntax` seems to be working on `v2` types](https://github.com/glimmerjs/glimmer-vm/tree/664a746b3baf82221c5874a759279d1afdd41e5b/packages/%40glimmer/syntax/lib/v2).

Rather than installing `@glimmer/syntax` as a development dependency to try to write the correct types, I opted to use `unknown` and ignore TS errors for the functions that I wrote and wrap `ember-template-recast`.